### PR TITLE
ACQ-1484 add management liability icon

### PR DESCRIPTION
**Pull request by katsiaryna-savchuk-itechart**
https://coverwallet.atlassian.net/browse/ACQ-1484

### CODE REVIEW

File: fonts/coverwallet.svg [hunk 1]
```diff
@@ -338,4 +338,5 @@
...
 </font></defs></svg>
```
Comment (MartaPalau) [inline fonts/coverwallet.svg:342]: eof
Why this icon is more complex now??

File: sass/lib/_coverwallet-font.scss [hunk 2]
```diff
@@ -1411,3 +1411,7 @@
 .cwicon-cannabis:before {
   content: "\ea4a";
 }
+
+.cwicon-insurance-type-management-liability:before {
+  content: "\ea4b";
+}
```
Comment (MartaPalau) [inline sass/lib/_coverwallet-font.scss:1417]: I think you need to add the html to add the icon in commented code https://github.com/coverwallet/styleguide/pull/48/files#diff-1b393107a3cd9a23f9b1b020fd9b0ac3R52

**Review by MartaPalau — COMMENTED** [review]
please check this PR, I  think there are some files missing https://github.com/coverwallet/styleguide/pull/48/files#diff-1b393107a3cd9a23f9b1b020fd9b0ac3R52

File: sass/lib/_coverwallet-font.scss (hunk 2 above)
Comment (katsiaryna-savchuk-itechart) [reply sass/lib/_coverwallet-font.scss:1418]: Checked locally, it worked even without html part as some of the other icons, but anyway added
Thanks!

**Review by MartaPalau — APPROVED** [review]

**Pull request merged**